rbd/rbd.go

```diff
@@ -809,6 +823,36 @@ func (image *Image) GetSnapshotNames() (snaps []SnapInfo, err error) {
 	return snaps[:len(snaps)-1], nil
 }
 
+func (image *Image) GetSnapId(snap_name string) (uint64 snapId, err error) {
+  snaps, err := image.GetSnapshotNames()
+
+  if err != nil {
+    return 0, err
+  }
+
+  for _, snap := range snaps {
+    if snap.Name == snap_name {
+      return snap.Id, nil
+    }
+  }
+  return 0, RbdErrorSnapNotFound
+}
+
+// int rbd_snap_get_timestamp(rbd_image_t image, uint64_t snap_id, struct timespec *timestamp);
+func (image *Image) GetSnapCreationTime(snap_name string) (timestamp time.Time, err error) {
+  var c_timespec C.struct_timespec
+
+  snapId, err := image.GetSnapId(snap_name)
+  if err != nil {
+    return time.Unix(0,0), err
+  }
+
+  ret := C.rbd_snap_get_timestamp(image.image, C.uint64_t(snapId), &c_timespec); ret < 0 {
```

Comment:
This appears to be missing an "if" at the start of the line, and a closing curly bracket for the block. Please run `go fmt` on your code - you'll save yourself the hassle of a rejected PR when the CI bot runs.

Reply:
Updated, sorry for the missing statements ;)